web/packages/teleport/src/Main/Main.tsx

```diff
@@ -189,14 +189,12 @@ export function Main(props: MainProps) {
     <FeaturesContextProvider value={features}>
       <TopBar
         CustomLogo={
-          props.topBarProps?.showPoweredByLogo
-            ? props.topBarProps.CustomLogo
-            : null
+          props.topBarProps?.CustomLogo ? props.topBarProps.CustomLogo : null
         }
       />
       <Wrapper>
         <MainContainer>
-          <Navigation />
+          <Navigation showPoweredByLogo={props.topBarProps.showPoweredByLogo} />
```

Comment:
What do you think of removing `showPoweredByLogo` from `TopBarProps`? 
I'm not sure if we need to configure `CustomLogo` and `showPoweredByLog` separately. I see that we even use the custom logo value as the source of truth for `showPoweredByLogo` in E.